src/controller/AutoCommissioner.cpp

```diff
@@ -274,6 +311,15 @@ CommissioningStage AutoCommissioner::GetNextCommissioningStageNetworkSetup(Commi
     return CommissioningStage::kCleanup;
 }
 
+CommissioningStage AutoCommissioner::GetNextStateAfterNetworkCommissioning()
```

Comment:
How does this work with non-concurrent commissioning?  I don't think it does...  This needs to be either before network commissioning or done over the operational network after network commissioning, right?

Reply:
Moving ICD registration steps to before network commissioning sounds good.

Just another question: so any steps after network commissioning steps should be done over op network and any steps over op network requires a op discovery?

For StayActive, I think it should be done after network commissioning and seems we cannot avoid a `kFindOperationalForStayActive`?